tests/tracing/test_misc.py

```diff
@@ -82,197 +58,31 @@ def test_transaction_data(sentry_init, capture_events):
     assert span_data.items() >= {"spanfoo": "spanbar"}.items()
 
 
-def test_start_transaction(sentry_init):
```

Comment:
most of the following tests are too unit-testy and don't add much or are obsolete.